src/opera_disp_tms/generate_frame_tile.py

```diff
@@ -37,10 +37,34 @@ def lat_string(lat):
     def lon_string(lon):
         return ('E' if lon >= 0 else 'W') + f'{abs(lon):03}'
 
-    bbox_str = f'{lat_string(bbox[1])}{lon_string(bbox[0])}_{lat_string(bbox[3])}{lon_string(bbox[2])}'
+    bbox_str = f'{lon_string(bbox[0])}{lat_string(bbox[3])}'
     return '_'.join([*parts, orbit_pass, bbox_str]).upper()
 
 
+def update_frame_geometry(frame: Frame, buffer_size: int = -3500) -> Frame:
```

Comment:
`buffer_frame_geometry` would be more expressive than `update`

Reply:
I'd also suggest `buffer_size_in_meters`  and/or capturing the units in the docstring

Reply:
Done!